shell/ci/auth/gcr.sh

```diff
@@ -2,6 +2,11 @@
 # Configures CircleCI docker authentication
```

Comment:
would be nice to no-op in this file if its running locally since docker login fails when running this locally via script-wrapper

Reply:
Good concern, planning on doing this later on with all the auth methods. Mostly just supporting a design that can do that in the future right now